src/test/java/io/confluent/idesidecar/restapi/application/SidecarInfoTest.java

```diff
@@ -65,23 +68,32 @@ Stream<DynamicTest> testCombinations() {
             "22.0413",
             OperatingSystemType.Linux,
             new SidecarInfo.VsCode("20.1.2", "1.2.3"),
-            null
+            null,
+            "Confluent-for-VSCode/v%s (https://confluent.io; support@confluent.io) sidecar/v%s (Linux/x86_64)".formatted(
+                vscodeVersion,
+                SidecarInfo.VERSION)
         ),
         new TestInputs(
             "Linux OS info with VS Code and version prefix",
             "Linux",
             "22.0413",
             OperatingSystemType.Linux,
             new SidecarInfo.VsCode("20.1.2", "1.2.3"),
-            "v"
+            "v",
+            "Confluent-for-VSCode/v%s (https://confluent.io; support@confluent.io) sidecar/v%s (Linux/x86_64)".formatted(
+                vscodeVersion,
+                SidecarInfo.VERSION)
```

Comment:
I think it would be better to statically specify what the output should be here rather than programmatically (or dynamically) specifying it. The reason is that we might end up exposing implementation details in our tests, and we ideally want to decouple the two. 

While writing this out, I realized that we might need to draw an exception for the tests-decoupled-from-implementation principle for baking in the `SidecarInfo.VERSION` into the expected user agent string. That said, it would be nice to be able to pass this into the `SidecarInfo` class rather than _always_ infer it from the `quarkus.application.version`. WDYT? cc @rhauch 

```suggestion
            "Confluent-for-VSCode/v20.1.2 (https://confluent.io; support@confluent.io) sidecar/v%s (Linux/x86_64)".formatted(SidecarInfo.VERSION)
```